### PR TITLE
ADD: Beta status badge

### PR DESCRIPTION
**Pull request by syedhamidali**
- [x] Closes #xxxx
- [x] Tests added
- [x] Changes are documented in `history.md`

### CODE REVIEW

File: tests/test_util.py [hunk 1]
```diff
@@ -264,3 +264,68 @@ def test_get_sweep_keys():
     dt["sneep_1"] = dt["sweep_1"]
     keys = util.get_sweep_keys(dt)
     assert keys == ["sweep_0", "sweep_1", "sweep_2", "sweep_3", "sweep_4", "sweep_5"]
+
```
Comment (kmuehlbauer) [inline tests/test_util.py:267]: This does belong to another PR, I believe. Please clean up, then we can merge.

**Comment by kmuehlbauer** [comment]
@syedhamidali As commented inline, please remove all additional code changes from the beta-badge PR, then we can merge.

Side note: On the openradar dev meeting before the ERAD, the was the conclusion to move fast to approach version 1.0.

File: tests/test_util.py (hunk 1 above)
Comment (syedhamidali) [reply tests/test_util.py:267]: @kmuehlbauer I am not sure I understood, do you mean I have to clean the code from the lines 264 to 267?

Comment (kmuehlbauer) [reply tests/test_util.py:267]: For the badge PR here, only the changes in history.md and README.md are necessary. The other changes have been creeping in from your other PR's I think.

Comment (syedhamidali) [reply tests/test_util.py:267]: Yes, you can first review the other two PRs, ' apply accessor` and `apply_to_sweeps`. You can ignore this PR for a while because these are minor changes here, and are not necessary at the moment. I could also close this PR if you want.

Comment (syedhamidali) [reply tests/test_util.py:267]: But, except for the changes in history and readme, this is the same as apply_to_sweeps PR.

Comment (kmuehlbauer) [reply tests/test_util.py:267]: I'd would wait for @mgrover1 to have a chance to have a look at the your PR's. We can merge this one immediately, once cleaned up. I'm fine either way.

Comment (syedhamidali) [reply tests/test_util.py:267]: I'm going to close this PR, since it is giving me conflicts with other PRs, and isn't needed at the moment.
